packages/tools/client-debugger/client-debugger-chrome-extension/src/react-components/ContainerHistoryView.tsx

```diff
@@ -69,6 +70,15 @@ export function ContainerHistoryView(props: ContainerHistoryProps): React.ReactE
 
 		messageRelay.on("message", messageHandler);
 
+		// Request state info for the newly specified containerId
```

Comment:
Fix a bug here

Reply:
Good catch😄 ! Can we just move this to a separate PR to keep each one focused?

Reply:
Sure. Removed it then